Automattic/jetpack
# Contact Form: Removing deprecated Google Drive Helper file, and jetpack_form_register_pattern function

Fixes https://github.com/Automattic/vulcan/issues/306

## Proposed changes:

* This PR cleans up after deprecations related to the Contact Form module in Jetpack (removing a deprecated function and the Jetpack_Google_Drive_Helper class, both of which have working alternatives in the Contact Form package).

### Other information:


## Jetpack product discussion

https://github.com/Automattic/vulcan/issues/306


## Does this pull request change what data or activity we track or use?

No.

## Testing instructions:

No functionality should be affected as we're removing deprecated files/functions.

However, we can confirm Contact Forms work as expected.

We can also confirm the Form inbox connects to Google drive as expected `/wp-admin/admin.php?page=jetpack-forms` - make sure you select 'Inbox' from the 'View' tab (top right of the dashboard). You'll need some form entries, then you can click on 'Export' and select 'Google Sheets'.

Nothing should be showing in error logs either.
